Libraries/LibWeb/HTML/HTMLHyperlinkElementUtils.cpp

```diff
@@ -206,7 +206,8 @@ String HTMLHyperlinkElementUtils::hostname() const
     URL::URL url(href());
 
     // 3. If url or url's host is null, return the empty string.
-    if (url.host().has<Empty>())
+    // FIXME: How can url be null here?
```

Comment:
The reason behind here is that the implementation doesn't follow spec - see: `HTMLHyperlinkElementUtils::hostname` as an example of how this should be done. `m_url` may be null.

Reply:
Thanks, done that in a later commit.